guides/source/active_record_basics.md

```diff
@@ -152,7 +154,7 @@ CREATE TABLE products (
 );
 ```
 
-Following the table schema above, you would be able to write code like the
+Schema above declares a table with two columns: `id` and `name`. Each row of this table represents a certain product with these two parameters. Thus, you would be able to write code like the
```

Comment:
Lines in documentation should be wrapped to 80 characters. See the above where you changed `SQL` to `MySQL` and how those are wrapped? That should be done on all new documentation.

Reply:
Done. )